app/assets/stylesheets/_simple-sidebar.scss

```diff
@@ -33,7 +33,12 @@
   list-style: none;
   font-size: 1em;
 
-  li {
+  .nav-item.active > .nav-link {
```

Comment:
Is this a Bootstrap class? If so, we should not add style for a class of a framework/lib, you could simply create a new class for that and add it to the element as well.